scripts/python/step_1.py

```diff
@@ -1,16 +1,21 @@
 ### Step 1 -- run this script locally
-### Create required env vars in the Circle-CI On-Prem project
-### These contain AWS keys and CircleCI API keys
+### Create required env vars in the Circle-CI On-Prem project, and trigger a CI job.
 ### WARNING: If env vars with the same names are already set for the project, 
 ### This script will OVERWRITE them! 
 
-from requests import post, exceptions
-from os import getenv
 from json import dumps, loads
+from git import Repo
+from os import listdir, path, getenv, makedirs, getcwd, chdir
+from requests import get, post, patch, exceptions
+from shutil import rmtree, copy
+from pathlib import Path as Path
+from functools import reduce
 
 org = getenv("MIGRATION_ORG")
-serverBaseURL = "https://circleci.{}.engineering".format(org)
 project = getenv("CIRCLE_PROJECT_REPONAME")
+serverBaseURL = "https://circleci.{}.engineering".format(org)
```

Comment:
This probably won't work for others that don't use the same TLD. I would break this out into it's own variable rather than replacing `org` here.

Reply:
what do you mean by TLD? :D I believe this is the standard convention for on-prem subdomains, but I guess if the org name is not == to the subdomain, that could be a problem! Will change this.

Reply:
I meant not everyone will have a `.engineering` domain in their org. Could be `.services` or whatever, so making the assumption here of `.engineering` would require a code change on this line for them.

Reply:
ahhh. TIL! thanks